app/src/main/kotlin/cz/covid19cz/erouska/utils/Auth.kt

```diff
@@ -23,8 +23,8 @@ object Auth: KoinComponent {
         return checkNotNull(auth.currentUser?.uid)
     }
 
-    fun getPhoneNumber(): String {
-        return checkNotNull(auth.currentUser?.phoneNumber)
+    fun getPhoneNumber(): String? {
+        return auth.currentUser?.phoneNumber
```

Comment:
Tady musíš vrátit to z sharedpreferences pokud nenajdeš ve firebase

Reply:
Je to napsané tak, že nově se hledá v SP a pokud se nenajde (uživatel updatnul ze staré verze) tak se vezme hodnota z Firebase. Pokud ani tady není tak se vrátí pomlčka.

Jinde se nově tato funkce getPhoneNumber() nepoužívá.

Reply:
Updated